test/dns_test.rb

```diff
@@ -106,5 +106,31 @@ module Yast
         end
       end
     end
+
+    describe ".IsHostLocal" do
```

Comment:
IMHO, this test is a bit incomplete.

See [this gist](https://gist.github.com/dgdavid/451c11d81211d8f98470cc35b08155f7) with a tested suggestion to improve it a little, which probably will need some changes if the chainable `stdout` method is finally used. [Here there is a modified version](https://gist.github.com/dgdavid/791127e83c4c9c7f1ff169c0941efa21).